blog-migration.js

```diff
@@ -286,15 +314,20 @@ export async function main(index = INDEX, source = SOURCE_CACHE, outputDir = OUT
             markdown = await loadOrFetchMarkdown(url, path);
         }
 
-        const pageReports = await handleMigration(markdown, entry, i, outputDir);
-        reports.push(...pageReports);
+        const [ pageReports, totalLinksReport ] = await handleMigration(markdown, entry, i, outputDir, entries);
+        reports.push(pageReports);
+        linkReport.push(totalLinksReport);
```

Comment:
If there is more than one block migration this wont work.

Reply:
seems probably better to just remove the link report then.